contracts/BC_fusion/StakeHub.sol

```diff
@@ -644,6 +654,13 @@ contract StakeHub is System, Initializable {
             uint256 newUnbondPeriod = value.bytesToUint256(32);
             if (newUnbondPeriod < 3 days) revert InvalidValue(key, value);
             unbondPeriod = newUnbondPeriod;
+        } else if (key.compareStrings("redelegateFeeRate")) {
+            if (value.length != 32) revert InvalidValue(key, value);
+            uint256 newRedelegateFeeRate = value.bytesToUint256(32);
+            if (newRedelegateFeeRate > REDELEGATE_FEE_RATE_BASE || newRedelegateFeeRate == 0) {
```

Comment:
==> `newRedelegateFeeRate > 100`

Reply:
I think we can allow `newRedelegateFeeRate == 0` in future.

Reply:
Done